packages/headless/src/controllers/breadcrumb-manager/headless-breadcrumb-manager.ts

```diff
@@ -135,34 +135,34 @@ export const buildBreadcrumbManager = (
     ...controller,
 
     /**
-     * @returns {BreadcrumbManagerState} The state of the `BreadcrumbManager` controller.
+     * The state of the `BreadcrumbManager` controller.
      */
     get state() {
       return {
         /**
-         * @returns {FacetBreadcrumb[]} The list of specific facet breadcrumbs.
+         * The list of specific facet breadcrumbs.
          */
         facetBreadcrumbs: getFacetBreadcrumbs(),
         /**
-         * @return {CategoryFacetBreadcrumb[]} The list of category facet breadcrumbs.
+         * The list of category facet breadcrumbs.
          */
         categoryFacetBreadcrumbs: getCategoryFacetBreadcrumbs(),
         /**
-         * @returns {NumericFacetBreadcrumb[]} The list of numeric facet breadcrumbs.
+         * The list of numeric facet breadcrumbs.
          */
         numericFacetBreadcrumbs: getNumericFacetBreadcrumbs(),
         /**
-         * @returns {DateFacetBreadcrumb[]} The list of date facet breadcrumbs.
+         * The list of date facet breadcrumbs.
          */
         dateFacetBreadcrumbs: getDateFacetBreadcrumbs(),
       };
     },
     /**
-     * Determines if there's any available breadcrumbs, or active facets.
+     * Determines if there are any available breadcrumbs (i.e., if there are any active facets).
      */
     hasBreadcrumbs,
```

Comment:
Hmm, this property would ideally be a derived property stored inside the `state` object. This pattern would be more consistent with the other controllers, and not require a developer to need to call the function.

From there, we can follow the same pattern for the jsdoc:
`true if there are facets with selected values, and false otherwise.`

```
get state() {
    return {
        hasBreadcrumb: hasBreadcrumb()
    }
}
```